### PR TITLE
📝 Add ES translation for home page

### PR DESCRIPTION
**Pull request by tropicadri**
ES new home translations

### CODE REVIEW

File: site/es/_data/home.yml [hunk 1]
```diff
@@ -1,78 +1,78 @@
...
     - src: ix://lifestyle/vim_window.jpg
-      alt: A Vim text editor with code.
+      alt: Un editor de texto Vim con código.
     - src: ix://home/2022/m100start.svg
-      alt: '"m100.start()" appears inside a repeating pattern of curly braces.'
+      alt: '"m100.start()" se muestra dentro de un patrón de paréntesis que se repiten'
 stats:
   modifiers:
     heading: Grow in good company.
     subtitle: Keep your app—and your audience—growing, across the Chrome OS community.
```
Comment (Snugug) [inline site/es/_data/home.yml:72]: Can you please take a look at this?

File: site/es/_data/home.yml [hunk 2]
```diff
@@ -1,78 +1,78 @@
...
   modifiers:
     variant: routing
 linux:
   content:
     title: |
-      Impressively versatile.
-      Exceptionally easy.
+      Impresionantemente versátil.
+      Excepcionalmente fácil.
     copy: Linux is seamlessly built into Chrome OS so you’ll always be ready whenever inspiration hits.
```
Comment (Snugug) [inline site/es/_data/home.yml:58]: Can you please take a look at this?

File: site/es/_data/home.yml [hunk 3]
```diff
@@ -1,78 +1,78 @@
...
-      alt: '"m100.start()" appears inside a repeating pattern of curly braces.'
+      alt: '"m100.start()" se muestra dentro de un patrón de paréntesis que se repiten'
 stats:
   modifiers:
     heading: Grow in good company.
     subtitle: Keep your app—and your audience—growing, across the Chrome OS community.
   items:
     - stat: +270%
-      desc: Desktop PWA installs, YoY
+      desc: Desktop PWA installs , YoY
```
Comment (Snugug) [inline site/es/_data/home.yml:75]: Can you please take a look at this?

File: site/es/_data/home.yml [hunk 4]
```diff
@@ -81,15 +81,15 @@ stats:
...
     - stat: 50M
-      desc: Students and educators using Chromebooks worldwide
+      desc: Estudiantes y educadores usando Chromebooks alrededor del mundo
       modifiers:
         shape: triangle
         scale: half
 caseStudies:
-  title: Get inspired.
+  title: Encuentra inspiración.
   subtitle: Explore how brands are unlocking more possibilities with Chrome OS.
```
Comment (Snugug) [inline site/es/_data/home.yml:90]: Can you please take a look at this?

File: site/es/_data/home.yml [hunk 5]
```diff
@@ -81,15 +81,15 @@ stats:
...
   cta:
-    text: All Case Studies
+    text: Casos de estudio
     url: stories
 m100:
```
Comment (Snugug) [inline site/es/_data/home.yml:94]: Can you please take a look at this? [This](https://github.com/chromeos/chromeos.dev/blob/main/site/en/_data/home.yml#L97-L107) is the current content for it

File: site/es/_data/home.yml [hunk 6]
```diff
@@ -153,7 +153,7 @@ developers:
           role: Head Engineer
       image:
         src: ix://lifestyle/developer_3.png
-        alt: Mike C. leaning against a desk.
+        alt: Mike C. recostado en un escritorio.
 community:
```
Comment (Snugug) [inline site/es/_data/home.yml:157]: Can you please take a look at this? [This](https://github.com/chromeos/chromeos.dev/blob/main/site/en/_data/home.yml#L132-L159) is the current content for it.

File: site/es/_data/home.yml [hunk 7]
```diff
@@ -106,26 +106,26 @@ posts:
...
+    subtitle: Descubre tendencias, eventos, y actualizaciones en nuestra comunidad.
     cta:
-      text: All Articles
+      text: Todos los artículos
       url: news
 educators:
   content:
-    title: Empower millions of students and educators.
+    title: Empodera a millones de estudiantes y educadores.
     copy: Learn how your app can power learning experiences for 50 million Chromebooks in the classrooms.
```
Comment (Snugug) [inline site/es/_data/home.yml:116]: Can you please take a look at this?

**Review by Snugug — CHANGES_REQUESTED** [review]
Thanks so much for doing this Adriana! I really appreciate it!

There look to be a few places where I think the merged Spanish file we had didn't match the English version, and some places where I think a translation may have been missed. I think I've pointed them all out in the doc, except for [footnotes](https://github.com/chromeos/chromeos.dev/blob/main/site/en/_data/home.yml#L196-L201), which are def in the "missing" category that I didn't get moved into this file (and [stats](https://github.com/chromeos/chromeos.dev/blob/main/site/en/_data/home.yml#L69-L90) will probably need to be updated to point to the right footnote too).

Again, thank you very much for your help here!

**Review by Snugug — CHANGES_REQUESTED** [review]
Thanks so much for this Adriana! I really really appreciate it!

It looks like there are two items that we added last-minute to the English page that didn't make it into the Spanish YAML. Can you take a look at them?

* [footnotes](https://github.com/chromeos/chromeos.dev/blob/main/site/en/_data/home.yml#L196-L201) and the matching [footnote number](https://github.com/chromeos/chromeos.dev/blob/main/site/en/_data/home.yml#L69-L90) in the stats
* The [quotes](https://github.com/chromeos/chromeos.dev/blob/main/site/en/_data/home.yml#L128-L159) in the developers section.

Again, thank you very much for your help here!

**Comment by tropicadri** [comment]
I double checked quotes and added footnotes. let me know if there is anything missing.

**Review by Snugug — APPROVED** [review]

**Pull request merged**